polaris-react/src/components/Text/Text.tsx

```diff
@@ -52,7 +52,6 @@ type Tone =
 type TextDecorationLine = 'line-through';
 
 const deprecatedVariants: {[V in Variant]?: Variant} = {
-  heading2xl: 'headingXl',
   heading3xl: 'headingXl',
```

Comment:
Should we map `heading3xl` to `heading2xl` now that we are keeping it?

Reply:
Super good catch! Yes! Thanks Sam 🙌🏻 🎉